Bio/SeqRecord.py

```diff
@@ -159,15 +162,15 @@ class SeqRecord:
 
     def __init__(
         self,
-        seq,
+        seq: Union["Seq", "MutableSeq", "UnknownSeq"],
```

Comment:
UnknownSeq is a subclass of Seq, so does ``Union["Seq", "MutableSeq"]`` suffice?

Reply:
It does, thanks!